### ISSUE TITLE
Built-in screen capture / screen shot functionality

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#1374]
## Updated description
![image](https://user-images.githubusercontent.com/89839714/131785718-0dc0529e-f917-4330-8149-d2d9733d0c41.png)

Include **web-capture** feature directly in **Brave** so that we don't have to install extensions to do that thing.

## Original issue description
Carried over from https://github.com/brave/browser-laptop/issues/12139

> Functions that are particularly useful includes:
> - it's easier to select parts of a page just by clicking on them (then being able to modify them)
> - capture a whole page
> - share the page

For reference, here are docs about the implementations in other browsers
- Firefox: https://screenshots.firefox.com/
- Vivaldi: https://help.vivaldi.com/article/capture-a-screenshot/

**Comment by Smooey** [comment]
Yes, please please pretty please include this into Brave. I'd never use another browser ever again, if this was included into Brave lol.

**Comment by eljuno** [comment]
+1 from Community https://community.brave.com/t/how-do-i-take-a-partial-screenshot-of-a-webpage-like-firefox-has/74335?u=eljuno

**Comment by NumDeP** [comment]
+1 Using third-party extensions are terribly inefficient and have more steps than one would wish for. Though I have to admit they've helped me quite a lot these past couple of years. 
I second @jamesray1 request of it to function exactly like in Mozilla. It's just fantastic, so fast, light and without a doubt the most efficient screen/webpage screen-capture ever.

Something that is well integrated within the browser like Mozilla's and doesn't unnecessarily consume memory in the background would be most ideal, in addition, it would be really good if repetitive tasks were somehow made easy. Most capture solutions allow either full, visible screen and a cropper feature, it would be really good if Brave enhanced the end solution to be even more efficient without a multitude of buttons by mitigating repetitive tasks for the user by remembering the structure of last screen capture of a certain page (site) taken, so when the button is pressed next, you don't have to make adjustments again and again.

Totally deserves a P3 label considering we're passed 1.0 now and this was sort of promised in 0.18.x Muon! :)

**Comment by xenithorb** [comment]
The Chrome webstore landscape for screenshot tools is a minefield of data collection and badly designed tools. I looked through maybe 20 to find one that was as simple and effective as Mozilla's in Firefox (as a refugee) and couldn't come across any. Most are developed by individuals, and tend to be of a dubious nature. 

The Chromium community at large would benefit greatly from a well-designed capture tool that could:

1. Collect full-page areas
1. Collect areas based on page elements
1. Basic annotations like arrows and text

This space is dying for an extension with a reputable company backing it like Brave. Hopefully this great team can find some cycles to get something going!

**Comment by bsclifton** [comment]
+1 from @clicky6 via https://github.com/brave/brave-browser/issues/17820

**Comment by Tonev** [comment]
@bsclifton 

Do you think the built-in Chromium screenshot functionality can be used as a first step into introducing such an option in some Brave menu?

1. Open `Developer tools`.
2. Use `Ctrl + Shift + P` to run a command.
3. Type `screenshot`.
4. Observe the provided options.

**Comment by kayzzen01** [comment]
> ## Updated description
> ![image](https://user-images.githubusercontent.com/89839714/131785718-0dc0529e-f917-4330-8149-d2d9733d0c41.png)
> 
> Include **web-capture** feature directly in **Brave** so that we don't have to install extensions to do that thing.
> 
> ## Original issue description
> Carried over from [brave/browser-laptop#12139](https://github.com/brave/browser-laptop/issues/12139)
> 
> > Functions that are particularly useful includes:
> > 
> > * it's easier to select parts of a page just by clicking on them (then being able to modify them)
> > * capture a whole page
> > * share the page
> 
> For reference, here are docs about the implementations in other browsers
> 
> * Firefox: https://screenshots.firefox.com/
> * Vivaldi: https://help.vivaldi.com/article/capture-a-screenshot/

Looking forward to this feature. I use this feature a lot in Firefox to make manuals.

**Comment by andrewaclear** [comment]
This would be nice but if you guys are just looking for a nice way to take screenshots, use http://scrnli.com/# extension which you can get from the chrome store [Scrnli Screenshot & Screen Video Recorder](https://chrome.google.com/webstore/detail/scrnli-screenshot-screen/ijejnggjjphlenbhmjhhgcdpehhacaal)

**Comment by nsklaus** [comment]
when i press "share" button on my gamepad (ds4),  i get a popup asking me if i want to allow screen capture for brave browser. this is annoying and i cannot find how to disable this. could anyone tell me how i can tell brave to not record anything ? i would like to disable it.

**Comment by bsclifton** [comment]
Closing as this is now solved 😄 

<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>S</kbd> for Windows 
<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>S</kbd> for macOS 

It's also available via hamburger menu 🎉

**Comment by plasmatech8** [comment]
Good to see. :)
It would be nice to also have a full-page screenshot button so we can include content down the page.
